### PR TITLE
ChemMaster UI tweaks

### PR DESCRIPTION
**Pull request by FluffyGhoster**
Tweaked the new chemmaster UI and added some sounds to its operations
The UI fonts are now smaller, the tabulations takes less space from the margins, and some sounds for the various buttons and operations are added

image

https://www.youtube.com/watch?v=k375NYDG0_c

Thanks to Ramke for the feedbacks on it

### CODE REVIEW

**Comment by FluffyGhoster** [comment]
!review

**Review by NonQueueingMatt — APPROVED** [review]

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 1]
```diff
@@ -45,6 +45,8 @@
 				qdel(src)
 				return
 
+#define CHEMMASTER_BOTTLE_SOUND playsound(src, 'sound/items/pickup/bottle.ogg', 75, 1)
+
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Machinery.dm:49]: The defines should be placed at the top unless there is a good reason not to do so.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 2]
```diff
@@ -153,12 +156,18 @@
 
 	return data
 
+#define CHEMMASTER_DISPENSE_SOUND playsound(src, 'sound/machines/reagent_dispense.ogg', 75, 1)
+#define CHEMMASTER_CHANGESETTINGS_SOUND playsound(src, 'sound/machines/slide_change.ogg', 75, 1)
+#define CHEMMASTER_SWITCH_SOUND playsound(src, 'sound/machines/switch1.ogg', 75, 1)
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Machinery.dm:161]: The defines should be placed at the top unless there is a good reason not to do so.

File: code/modules/reagents/Chemistry-Machinery.dm [hunk 3]
```diff
@@ -265,17 +278,25 @@
...
 	else if(action == "bottle_sprite")
 		bottlesprite = sanitizeSafe(params["bottle_sprite"])
+		CHEMMASTER_CHANGESETTINGS_SOUND
 		return TRUE
 	return TRUE
 
+#undef CHEMMASTER_DISPENSE_SOUND
+#undef CHEMMASTER_CHANGESETTINGS_SOUND
+#undef CHEMMASTER_SWITCH_SOUND
+#undef CHEMMASTER_BOTTLE_SOUND
```
Comment (Arrow768) [inline code/modules/reagents/Chemistry-Machinery.dm:298]: The undefs should be placed at the bottom of the file unless there is a good reason not to do so.

**Review by Arrow768 — CHANGES_REQUESTED** [review]

**Review by Arrow768 — APPROVED** [review]

**Pull request merged**